### DISCUSSION TITLE
Color Swatch

### DISCUSSION THREAD

**Discussion by Navjeet** [kondasoft/ks-bootshop#77] [Ideas] [answered]
Hi, @alvinkonda 

First of all, Thank you so much for the great theme.

Can we have Color Swatch option on the product page and Multiple variant images for single variant option?

Thank you.

**Comment by alvinkonda** [comment]
Our goal for this theme is to keep it simple and basic, with only the necessary features needed. Those 2 features you are mentioning are above the basic needs, and unfortunately, we do not plan to add those to this theme. But of course, we can always help you with custom development work if those features are important for you and you are not able to code them yourself. Let me know

**Reply by Navjeet** [reply]
Hi @alvinkonda 

Thank you for your reply.

I have a one more question.

May I use this theme as a starter base to develop new theme and add new feature to sell on platform.

Looking forward to your reply.

Thank you.

**Comment by alvinkonda** [comment, marked as answer]
Yes of course, but if the business takes off, don't forget to send some coffee :)

**Reply by Navjeet** [reply]
Hi @alvinkonda 

Thanks for your reply.

Yes sure, I will.

Thank you :)
